examples/blob/main.py

```diff
@@ -0,0 +1,22 @@
+from pyteal import *
```

Comment:
Even for the examples, I recommend avoiding * imports... but that's not as important as in @jasonpaulos 's comment